src/RawModelExportList.tsx

```diff
@@ -17,43 +16,37 @@ function RawModelExportList({ deployment }: { deployment: Deployment }) {
   const { versions } = useGetModelVersions(model?.uuid)
 
   return (
-    <>
+    <Box>
       {versions &&
-        versions.map(
-          (version: any) =>
-            version.metadata?.buildOptions?.exportRawModel && (
-              <Box key={version.version}>
-                <Box sx={{ p: 1 }}>
-                  <Box sx={{ p: 2 }}>
-                    <Typography variant='h4'>Version: {version.version}</Typography>
-                  </Box>
-                  <Stack spacing={2} direction='row' sx={{ p: 1 }}>
-                    <Button
-                      variant='contained'
-                      href={`/api/v1/deployment/${deployment.uuid}/version/${version.version}/raw/code`}
-                      target='_blank'
-                      data-test='downloadCodeFile'
-                    >
-                      Download code file
-                    </Button>
-                    <Button
-                      variant='contained'
-                      href={`/api/v1/deployment/${deployment.uuid}/version/${version.version}/raw/binary`}
-                      target='_blank'
-                      data-test='downloadBinaryFile'
-                    >
-                      Download binary file
-                    </Button>
-                  </Stack>
-                </Box>
-                <Divider orientation='horizontal' />
+        versions.map((version: any) => (
+          <Box key={version.version}>
+            <Box sx={{ p: 1 }}>
+              <Box sx={{ p: 2 }}>
+                <Typography variant='h4'>Version: {version.version}</Typography>
               </Box>
-            )
-        )}
-      {versions && versions.filter((e) => e.metadata.buildOptions?.exportRawModel).length === 0 && (
-        <EmptyBlob text='No exportable versions' />
```

Comment:
You still need the empty blob check.

Reply:
I didn't think it would be possible for a deployment to _not_ have any versions but can add it back in.